src/http-servers/plain-text-server.js

```diff
@@ -0,0 +1,13 @@
+import http from 'http';
+const { log } = console;
+
+const port = process.env.port || 3000;
+//create a server object:
+http.createServer((req, res) => {
+   
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+
+    res.write('Hello World');
```

Comment:
the same here - please do `res.end(text)`

Reply:
Thanks, updated code.